### PR TITLE
Nfts: Get Token Nft Infos Query

### PR DESCRIPTION
**Pull request by daniel-danev-limechain**
**Description**:

- Added GetTokenNftInfosQuery realization which uses the FCOTMR structure and retrieves information for all of the nfts created from token, according to HIP-17.
- Return TOKEN_WAS_DELETED response when NON_FUNGIBLE_UNIQUE token is deleted.
- Unit tests were added for code coverage.
- Various E2E scenarios were added as well.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:
Already merged with master

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Daniel-K-Ivanov — DISMISSED** [review]
LGTM

File: test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java [hunk 1]
```diff
@@ -206,6 +206,7 @@ private AccountID randomNodeFrom(HapiApiSpec spec) {
 				updateStateOf(spec);
 			}
 		} catch (Throwable t) {
+			t.printStackTrace();
```
Comment (tinker-michaelj) [inline test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java:209]: As noted on #1751, this blows up our regression suite. 😝

Comment (daniel-danev-limechain) [reply test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java:209]: Seems like merge leftover from refactored nfts touchups. I've removed, so regression suite can commence

**Review by Daniel-K-Ivanov — APPROVED** [review]
LGTM

**Review by qnswirlds — APPROVED** [review]
LGTM. Minor changes will be addressed in #1755

**Comment by tinker-michaelj** [comment]
Merged via #1755

**Pull request closed without merging**